src/Core/Content/Property/PropertyGroupDefinition.php

```diff
@@ -76,6 +79,7 @@ protected function defineFields(): FieldCollection
             (new StringField('display_type', 'displayType'))->setFlags(new Required()),
             (new StringField('sorting_type', 'sortingType'))->setFlags(new Required()),
             new BoolField('filterable', 'filterable'),
+            new BoolField('visible_on_detail', 'visibleOnDetail'),
```

Comment:
we should not shorten the field unneeded. does something speaks agains `visible_on_product_detail_page` ?